api/billingd/main.go

```diff
@@ -48,6 +48,10 @@ var (
 				Key:      "addr.gateway.host",
 				DefValue: "/ip4/127.0.0.1/tcp/8010",
 			},
+			"addrAnalyticsApi": {
+				Key:      "addr.analytics_api",
+				DefValue: "",
+			},
```

Comment:
Any user of the analytics client needs to be configured to create the analytics client, so here in the billing daemon, we add the information we need to create the analytics client. You'll see this elsewhere in hub as well.

Reply:
@textileben here is a new config option for `billingd`